.storybook/main.js

```diff
@@ -22,9 +22,11 @@ module.exports = {
       '@babel/preset-react'
     );
 
-    // Contrary to storybook's documentation, it does not appear to be loading
-    // our root .babelrc, so we customize the default config's react preset
+    // Contrary to storybook's documentation, it did not appear to be loading
+    // our root .babelrc, so we customized the default config's react preset
     // importSource.
+    // Note that we don't even have a root .babelrc anymore since we now build
+    // with tsc.
```

Comment:
Updated to reflect the fact that we aren't using babel anymore to build.

Reply:
does this comment mean anything anymore? could it just be removed entirely?

Reply:
Yeah, I suppose it's not that useful anymore; the commit with the actual config is sufficient.  I'll remove it.